### DISCUSSION TITLE
True diversity with 2 receivers and 1 transmitter

### DISCUSSION THREAD

**Discussion by mstrens** [ExpressLRS/ExpressLRS#2565] [Q&A]
I know the Gemini concept but I would like to use some hardware I already have .

I have currently 2 receivers betafpv nano 2.4g (having no diversity antenna) and one transmitter Happymodel pro Tx 2.4g.
I also have a module (openXsensor) that can generate PWM signal based on the frames coming from 2 different elrs receivers. The module can get ELRS frames on 2 different inputs so it can be connected to the Tx pins from 2 different elrs receivers. It generates PWM signals if it get valid frames from at least one of them.
This module can also be connected to the Rx pin of ONE of the receivers in order to provide telemetry data.

So the Idea would be that both receivers get the signal from the transmitter but that only one send telemetry data.

ELRS let to setup the telemetry rate in the handset with a Lua script but this setup will be taken into account by both receivers.
So both receivers would normally send simultaneously telemetry frames to the handset and there would be some mismatch.

I imagine I have to make a small change to the ELRS firmware uploaded in one of the 2 receivers so that this receiver never send telemetry frames.

Do you think this is possible?
Perhaps it has already be done?
Thanks for your comment/help.

**Comment by mha1** [comment]
just use the WebUI to turn off telemetry on one of the receivers
